### PR TITLE
Corporate devices in the user profile

### LINKED ISSUE #142: Сorporate devices in the user profile

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Feature

## Related issues

Resolve #142

### CODE REVIEW

File: src/components/users/UserContacts.tsx [hunk 1]
```diff
@@ -25,22 +32,29 @@ const StyledLink = styled(Link)`
...
+    font-size: 14px;
+`;
+
 const StyledCard = styled.div`
     height: 100%;
 `;
 
 type UserContactsProps = {
     user: User | undefined;
+    visible?: boolean;
```
Comment (tsumo) [inline src/components/users/UserContacts.tsx:47]: What's the purpose of this prop? To hide devices? Why is it needed?

File: src/components/users/UserProfile.tsx
Comment (tsumo) [inline src/components/users/UserProfile.tsx:1]: Lots of unused imports and styled components

File: src/components/users/UserProfile.tsx [hunk 2]
```diff
@@ -132,13 +133,19 @@ export const UserProfile = () => {
     const orgStructureGroup = groupMemberships.filter(({ isOrgGroup }) => isOrgGroup)[0];
 
     const teams = groupMemberships.filter(({ isOrgGroup }) => !isOrgGroup);
+    const bio = user?.bio;
+    const name = user?.fullName;
+    const source = user?.source;
+    console.log('bio', bio);
```
Comment (tsumo) [inline src/components/users/UserProfile.tsx:139]: Dev logs

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/users/UserContacts.tsx (hunk 1 above)
Comment (Katrin-kudryash) [reply src/components/users/UserContacts.tsx:47]: Because the devices aren`t used in the popup

Comment (tsumo) [reply src/components/users/UserContacts.tsx:47]: `hideDevices` would be a better suited name for a prop. Or maybe `popupMode`

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/users/UserContacts.tsx [hunk 3]
```diff
@@ -72,7 +85,40 @@ export const UserContacts = ({ user }: UserContactsProps) => {
...
+                    {/* TODO: Link to add to the teams */}
+                    <InlineTrigger
+                        icon={<IconPlusCircleOutline noWrap size="s" />}
+                        text={'Add link'}
+                        onClick={() => {}}
+                    />
+                </StyledInfo>
+
+                <StyledInfo>
+                    {hideDevices && (
```
Comment (tsumo) [inline src/components/users/UserContacts.tsx:97]: Should be changed to `!hideDevices` or renamed to `showDevices`

**Review by tsumo — CHANGES_REQUESTED** [review]

**Review by tsumo — APPROVED** [review]

**Review by 9teen90nine — APPROVED** [review]

**Pull request merged**